tests/xlsx/xlsx_import.test.ts

```diff
@@ -319,6 +320,86 @@ describe("Import xlsx data", () => {
     expect((cf.rule as CellIsRule).values).toEqual(values);
   });
 
+  test.each([
+    ["number", "A2"],
+    ["decimal", "B2"],
+    ["list", "C2"],
+    ["list", "D2"],
+    ["date", "E2"],
+    ["time", "F2"],
+    ["textLength", "G2"],
+    ["date", "H2"],
+    ["custom", "I2"],
+  ])("Can import data validation rule %s", (ruleType, cellXc) => {
+    const testSheet = getWorkbookSheet("jestDataValidations", convertedData)!;
+    const dvRule = getDataValidationBeginningAt(cellXc, testSheet);
+    switch (ruleType) {
+      case "number":
+      case "time":
+      case "textLength":
+        // Unsupported data validation type 'textLength'
+        expect(dvRule).toBeUndefined();
+        break;
+      case "decimal":
+        expect(dvRule).toMatchObject({
```

Comment:
if you do a `.each` then switch/case on the value of the "each", then it should just be different tests. Even if it means repeating some setup lines